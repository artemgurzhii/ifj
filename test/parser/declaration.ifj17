let a, b: int
let a = 5
let a: string = "foo"