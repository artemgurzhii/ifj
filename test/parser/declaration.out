(let
  (decl
    (id a)
    (id b)
    : (id int)))

(let
  (decl
    (id a))
   = (int 5))

(let
  (decl
    (id a)
    : (id string))
   = (string 'foo'))
